src/v2/services/implementations/PolkadotWalletService.ts

```diff
@@ -77,6 +77,14 @@ export class PolkadotWalletService extends WalletService implements IWalletServi
           tip = tip ? ethers.utils.parseEther(tip).toString() : '1';
         }
 
+        const useableBalance = await this.assetsRepository.getNativeBalance(senderAddress);
+        const message = 'Minimum balance on the network is 0.01';
+        if (Number(useableBalance) < Number(100000000000000000)) {
```

Comment:
Perhaps the number is too big to calculate. Let's convert the decimals before compare these values.  Maybe you can do something like this:
```
 // REQUIRED_MINIMUM_BALANCE: 0.05
  const isBalanceEnough =
          Number(ethers.utils.formatEther(useableBalance)) > REQUIRED_MINIMUM_BALANCE;
 if (isBalanceEnough) {
   // Logic goes here
        }
```

Also, let's define the REQUIRED_MINIMUM_BALANCE in somewhere (perhaps at `modules/extrinsic/index.ts` or at `modules/toast/index.ts`).

<img width="1314" alt="image" src="https://github.com/AstarNetwork/astar-apps/assets/92044428/a9117d83-5073-4e48-abe0-4ecab5ddc35f">